keyboards/handwired/dygma/raise/raise.c

```diff
@@ -0,0 +1,145 @@
+/* Copyright 2018-2021 James Laird-Wah, Islam Sharabash
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "raise.h"
+
+void keyboard_post_init_kb(void) {
+    set_all_leds_to(0, 0, 0);
+    keyboard_post_init_user();
+}
+
+
+// "led_map" is taken from kaleidoscope
+// LHK = Left Hand Keys
+// LPH = "Leds per hand", which isn't _actually_ the number of leds, but instead
+// is just used for indexing
+#define LHK 33
+#define LPH 72
+
+#ifdef RGB_MATRIX_ENABLE
+#if defined (ansi)
```

Comment:
Use separate revision folders instead of doing this. It won't work with the Configurator.

Reply:
This was a pattern I copied from hnah40rgb. What do you mean by using another revision?

Reply:
`dygma/raise/ansi` and `dygma/raise/iso` etc.

Reply:
done